src/app/components/OptimoPromos/Link/index.jsx

```diff
@@ -0,0 +1,34 @@
+import React from 'react';
+import { node, string, shape } from 'prop-types';
+import useCombinedClickTrackerHandler from '#containers/StoryPromo/useCombinedClickTrackerHandler';
+import StyledLink from './index.styles';
+
+const Link = ({ className, children, toLink, a11yId, eventTrackingData }) => {
```

Comment:
I think `to` is a widely adopted prop name rather than `toLink`

Reply:
Done. I thought that it was a reserved keyword since the component was breaking when I used it... but now it is working fine...

Reply:
nvm, now I remember why I went for toLink, AMP validation is failing, I will have a better look at this once I fix all the easy stuff:
<img width="724" alt="Screenshot 2022-07-18 at 11 50 04" src="https://user-images.githubusercontent.com/90621252/179496764-a4a0b05a-721d-4479-8db1-020f6f626357.png">

Reply:
it sounds like you've changed the `href` html attribute to `to` e.g.

```jsx
to={to}
```

it should be like this

```jsx
href={to}
```